qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -141,9 +141,9 @@ def get_expected_result(self, expected_result, value, trial, flag_str=None):
         # information.
         if ((not NO_BATCHING and
              ("custom" not in trial)) or ("graphdef" in trial) or
-            ("plan" in trial) or ("onnx" in trial)) or ("libtorch" in trial):
+            ("plan" in trial) or ("onnx" in trial)) or ("libtorch" in trial) or ("python" in trial):
```

Comment:
I was wondering whether we can implement the full accumulator similar to the tensorflow model for Python backend instead of using an identity model?

Reply:
Full accumulator present in latest check in. These lines were removed. Closing this thread.